mlrun/model_monitoring/stream_processing.py

```diff
@@ -145,6 +145,20 @@ def apply_monitoring_serving_graph(
             fn.set_topology(mlrun.serving.states.StepKinds.flow, engine="async"),
         )
 
+        # forward back complete events to controller
+        graph.add_step(
+            "storey.Filter",
+            "FilterBatchComplete",
+            _fn="(event.get('kind') == 'batch_complete')",
+        )
+
+        graph.add_step(
+            "Delay",
+            name="Delay",
```

Comment:
just a recommendation - maybe name it "BatchDelay" just to make it clear as we may introduce additional “delay” steps in the future.

```suggestion
            name="BatchDelay",
```